spec/shortener/shortened_url_record_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe Shortener::ShortenedUrl do
+
+  def generate_shortened_link(link, expiration_time)
+    Shortener::ShortenedUrl.generate(link, expires_at: expiration_time)
+  end
+
+  LINKS = ['https://stackoverflow.com',
+           'https://github.com',
+           'https://twitter.com',
+           'https://reddit.com']
+
+  it "returns a default redirect" do
```

Comment:
We want to test whether the shortened link appears in/out of the scope based on the set expiry date. We should test:
1. when there is no expiry date
2. when there is an expiry date that has already past
3. when there is an expiry date that is still in the future

We also want to test the scenario that caused you to reveal this edgecase - e.g. that having an alternate timezone causes the logic to fail.

Without doing some poking around, I am not sure the bug will cause problems whenever the system is not UTC, or perhaps it only occurs when the db server timezone is different to the application timezone. 

If you could determine the scenario when this breakage occurs, and then craft some tests to simulate that scenario, it will demonstrate that your fix has done the job.